doc/api/events.md

```diff
@@ -142,7 +142,7 @@ myEmitter.emit('error', new Error('whoops!'));
 
 To guard against crashing the Node.js process, a listener can be registered
 on the [`process` object's `uncaughtException` event][] or the [`domain`][] module
-can be used. (_Note, however, that the `domain` module has been deprecated_)
+can be used. _(Note, however, that the `domain` module has been deprecated.)_
```

Comment:
I am not a native speaker, but as for me, the punctuation around "however" is pretty valid here as is.